lms/envs/common.py

```diff
@@ -961,7 +961,7 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 
 # Locale/Internationalization
 TIME_ZONE = 'America/New_York'  # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
-LANGUAGE_CODE = 'en'  # http://www.i18nguy.com/unicode/language-identifiers.html
+LANGUAGE_CODE = 'ro'  # http://www.i18nguy.com/unicode/language-identifiers.html
```

Comment:
Do we really want to make Romanian as fallback language?

Reply:
To enable a lang,  adding `ro` in `/admin/dark_lang/darklangconfig/` to the released languages list will be enough
(then, a user will be able to choose it in account settings)

Docs https://github.com/edx/edx-platform/wiki/Internationalization-and-localization#releasing-a-language

Reply:
That's Romanian site for romanian students. Why do you think it's a bad idea to set romaninan lang as a default one?

Reply:
It's not stated so in task requirements. Enabling a new language and setting a new default look like two different features to me.

Reply:
If I'm missing something and we really need to set `ro` as a new default, here's how we did it in other projects: 
- https://github.com/raccoongang/edx-platform/pull/1982 (and we updated LANGUAGE_LANG updated in lms.env.json; just another approach).
- similar PR: https://github.com/raccoongang/edx-platform/pull/1265

If you want to take a look:) 

Updating only the Django setting didn't seem to really work in all use cases (we had to preserve user preferences feature).